### PR TITLE
Fix bug in "ignore-response-error-level" parameter

### PR DESCRIPTION
**Pull request by ebadyano**
Parameter cannot be hyphenated

### CODE REVIEW

File: geopointshape/challenges/default.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
           "operation": "index-append",
           "warmup-time-period": 120,
           "clients": {{bulk_indexing_clients | default(8)}},
-          "ignore-response-error-level": "{{error-level | default('non-fatal')}}"
+          "ignore-response-error-level": "{ | default('non-fatal')}}"
```
Comment (DJRickyB) [inline geopointshape/challenges/default.json:31]:
```suggestion
          "ignore-response-error-level": "{{error_level | default('non-fatal')}}"
```

File: geopointshape/challenges/default.json [hunk 2]
```diff
@@ -100,7 +100,7 @@
...
```
Comment (DJRickyB) [inline geopointshape/challenges/default.json:103]:
```suggestion
          "ignore-response-error-level": "{{error_level | default('non-fatal')}}"
```

File: geopointshape/challenges/default.json [hunk 3]
```diff
@@ -164,7 +164,7 @@
           "operation": "index-update",
           "warmup-time-period": 120,
           "clients": {{bulk_indexing_clients | default(8)}},
-          "ignore-response-error-level": "{{error-level | default('non-fatal')}}"
+          "ignore-response-error-level": "{ | default('non-fatal')}}"
```
Comment (DJRickyB) [inline geopointshape/challenges/default.json:167]:
```suggestion
          "ignore-response-error-level": "{{error_level | default('non-fatal')}}"
```

**Review by DJRickyB — CHANGES_REQUESTED** [review]
couple inadvertent deletions looks like

File: geopointshape/challenges/default.json (hunk 1 above)
Comment (ebadyano) [reply geopointshape/challenges/default.json]: uh, good catch, thank you

**Review by DJRickyB — APPROVED** [review]
LEBTM

**Pull request merged**